app/mutator.py

```diff
@@ -63,17 +66,45 @@ def __init__(self):
     self.savedAtRedoIndex = 0
     self.shouldReparse = False
 
-  def compoundChangeBegin(self):
-    assert self.__compoundChange is None
+  def compoundChangeReset(self):
```

Comment:
I'm trying to think of another name for ```compoundChangeReset```. It does 'reset', but it does much more before that. Maybe 
```compoundChangeRefresh```, or 
```compoundChangeRenew```, or 
```compoundChangeCycle```, or
```compoundChangePush```, or 
```compoundChangeAgain```.
Do any of those sound ok, or something else?

Reply:
I like compoundChangePush. Unless you want more descriptive names that can be longish like
compoundChangePushAndReset or something along those lines.